### ISSUE TITLE
accessible html tables

### ISSUE THREAD

**Issue by davidstenger** [18F/web-design-standards#905]
I work for FCIC/OCSIT as an Accessible developer/designer an noticed a few issues in your responsive tables. 

The “scope” attribute on the “td” element is obsolete. Use the “scope” attribute on a “th” element instead. All <th> elements should generally always have a scope attribute. While screen readers may correctly guess whether a header is a column header or a row header based on the table layout, assigning a scope makes this unambiguous.
```<tbody>
<tr>
<td scope="row"> Declaration of Independence</td> [this TD attribute should be a TH scope="row"]
<td>Statement adopted by the Continental Congress declaring independence from the British Empire.</td>```


Below is an example that we us for tables.
example: http://promotions.usa.gov/platform/register-vote.html

**Comment by maya** [comment]
Thanks for catching this! We're working on a fix.

**Comment by maya** [comment]
Fixed in #907. 

Thanks @davidstenger! :tada:
